package/yast2-services-manager.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Aug 22 16:30:11 CEST 2019 - schubi@suse.de
+
+- Using rb_default_ruby_abi tag in the spec file in order to
+  handle several ruby versions (bsc#1146403).
+- 4.2.3
```

Comment:
It seems that there have been some errors in the past regarding the versioning. fixed.

Reply:
If I understand it correctly, we had the version outdated in the .spec file, isn't it?

Apart from that, I see that the `4.2.1` was skipped.